scholly/service/core/push.js

```diff
@@ -146,13 +152,13 @@ async function createPagesSite(ghUsername, ghToken, repoName) {
       data: pagesBody,
     });
     if (createPagesResponse.status === 201) {
-      console.log('Pages site successfully created');
+      logger.log('Pages site successfully created');
     } else {
       // see https://docs.github.com/en/rest/reference/repos#create-a-github-pages-site
-      console.log(createPagesResponse.data);
+      logger.log(createPagesResponse.data);
```

Comment:
Should we make this a debug level log, if printing out the response isn't strictly necessary?